Source/Charts/Renderers/PieChartRenderer.swift

```diff
@@ -358,10 +359,10 @@ open class PieChartRenderer: NSObject, DataRenderer
                 let sliceXBase = cos(transformedAngle.DEG2RAD)
                 let sliceYBase = sin(transformedAngle.DEG2RAD)
                 
-                let drawXOutside = drawEntryLabels && xValuePosition == .outsideSlice
-                let drawYOutside = drawValues && yValuePosition == .outsideSlice
-                let drawXInside = drawEntryLabels && xValuePosition == .insideSlice
-                let drawYInside = drawValues && yValuePosition == .insideSlice
+                let drawXOutside = sliceAngle > drawSliceTextMinimumAngle && drawEntryLabels && xValuePosition == .outsideSlice
```

Comment:
In an effort to make this a little cleaner, do 
```
guard sliceAngle > drawSliceTextMinimumAngle else { continue }
```

and then you don't need to add it to each line.

Reply:
The way the core loop is written, `continue` breaks the `xIndex` handling later in the loop. Looking over the code, I can't tell if the `xIndex` actually works correctly in cases where there are multiple data sets, but I don't understand it well enough to tweak it. This loop feels like it's complicated enough that I don't want to do more than minimum necessary changes.

I'm also not sure if we want to disable drawing the icons, which this change would do as well.

Happy to do something like:

```
var drawXOutside = drawEntryLabels && xValuePosition == .outsideSlice
var drawYOutside = drawValues && yValuePosition == .outsideSlice
var drawXInside = drawEntryLabels && xValuePosition == .insideSlice
var drawYInside = drawValues && yValuePosition == .insideSlice
if sliceAngle <= sliceTextDrawingThreshold {
    drawXOutside = false
    drawYOutside = false
    drawXInside = false
    drawYInside = false
}
```